tests/test_ace_fitting.py

```diff
@@ -49,25 +23,39 @@ def test_ace_fit_dry_run(request, tmp_path, monkeypatch, run_dir='run_dir'):
     (tmp_path / run_dir).mkdir()
 
     fit_config_file = os.path.join(os.path.dirname(request.fspath), 'assets', 'B_DFT_data.xyz')
```

Comment:
Since we're using `pathlib.Path` elsewhere, can we get rid of all the `os.path` syntax?